gateway/src/api/auth_layer.rs

```diff
@@ -117,15 +119,15 @@ where
         }
 
         // If /users/reset-api-key is called, invalidate the cached JWT.
-        if req.uri().path() == "/users/reset-api-key" {
+        if req_path == "/users/reset-api-key" {
             if let Some((cache_key, _)) =
                 cache_key_and_token_req(&req, self.gateway_admin_key.as_str())
             {
                 self.cache_manager.invalidate(&cache_key);
             };
         }
 
-        if req.uri().path().starts_with("/users") {
+        if req_path.starts_with("/users") || req_path.starts_with("/subscribe") {
```

Comment:
Could subscribe also be behind `/users`?

Reply:
Let me check...

Reply:
Done